reana_client/cli/secrets.py

```diff
@@ -0,0 +1,196 @@
+                    fg='red'),
+                err=True)
+    for path in from_file:
+        try:
+            with open(path, 'rb') as file:
+                secret = {
+                    file.name: {
+                        'value': base64.b64encode(
+                            file.read()).decode('utf-8'),
+                        'type': 'file'
+                    }
+                }
+                secrets.update(secret)
+        except FileNotFoundError as e:
+            logging.debug(traceback.format_exc())
+            logging.debug(str(e))
+            click.echo(
+                click.style(
+                    'File {0} could not be uploaded: {0} does not exist.'.
+                    format(path),
+                    fg='red'),
+                err=True)
+    try:
+        add_secrets(secrets, overwrite, access_token)
+    except Exception as e:
+        logging.debug(traceback.format_exc())
+        logging.debug(str(e))
+        click.echo(
+            click.style(
+                'Something went wrong while uploading secrets',
+                fg='red'),
+            err=True)
+    else:
+        click.echo(
+            click.style('Secrets {} were successfully uploaded.'.format(
+                ', '.joint(secrets.keys())),
+                fg='green')
+        )
+
+
+@secrets_group.command()
+@add_access_token_options
+@click.argument('secrets', type=str, nargs=-1)
+def secrets_delete(secrets, access_token):  # noqa: D301
+    """Delete users secrets with a specified names.
+
+     Examples: \n
+    \t $ reana-client secrets-delete PASSWORD
+    """
+    if not access_token:
+        click.echo(
+            click.style(ERROR_MESSAGES['missing_access_token'], fg='red'),
+            err=True)
+        sys.exit(1)
+
+    try:
+        response = delete_secrets(secrets, access_token)
+    except Exception as e:
+        logging.debug(traceback.format_exc())
+        logging.debug(str(e))
+        click.echo(
+            click.style(
+                'Something went wrong while deleting secrets',
+                fg='red'),
+            err=True)
+    else:
+        click.echo(
+            click.style('Secret(s): {} were successfully deleted.'.format(
+                ', '.joint(secrets)),
+                fg='green'))
+
+
+@secrets_group.command()
+@add_access_token_options
+def secrets_list(access_token):  # noqa: D301
+    """List users secrets.
+
+    Examples: \n
+    \t $ reana-client secrets-list
+    """
+    try:
+        _url = current_rs_api_client.swagger_spec.api_url
+    except MissingAPIClientConfiguration as e:
+        click.secho(
+            'REANA client is not connected to any REANA cluster.',
+            fg='red', err=True
+        )
+        sys.exit(1)
+
+    if not access_token:
+        click.echo(
+            click.style(ERROR_MESSAGES['missing_access_token'], fg='red'),
+            err=True)
+        sys.exit(1)
+
+    try:
+        response = list_secrets(access_token)
```

Comment:
I would change the naming here and everywhere else we use `response` when we call the `api_client` and rather use a more descriptive name such as `secrets_list`, since the `list_secrets(access_token)` method returns an actual list and not an "HTTP response", what do you think?